### PR TITLE
[4.0] Update build script

### PR DESCRIPTION
**Pull request by brianteeman**
Removes reference to non-existent files

code review

### CODE REVIEW

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on bb260304ee52190c8a033aa889947c3ded1652d1<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34440">issues.joomla.org/tracker/joomla-cms/34440</a>.</sub>

File: build/build.php [hunk 1]
```diff
@@ -370,9 +370,6 @@ function clean_composer(string $dir)
 	// Media Manager Node Assets
 	'administrator/components/com_media/webpack.config.js',
 	'administrator/components/com_media/resources',
-	// Remove the testing sample data from all packages
-	'installation/sql/mysql/sample_testing.sql',
```
Comment (wilsonge) [inline build/build.php:374]: This should be replaced with the testing installation plugin data i guess? Can't remember the decisions on that off the top of my head

Comment (brianteeman) [reply build/build.php:374]: you mean the testing sampledata plugin?

if so then thats already not in the release (unless you built the RC by hand)

Comment (brianteeman) [reply build/build.php:374]: never mind - its already listed and removed
https://github.com/joomla/joomla-cms/blob/bb260304ee52190c8a033aa889947c3ded1652d1/build/build.php#L130

**Comment by brianteeman** [comment]
someone more skilled than me should probably check that there are not other files that should be added to the "do not package" list

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
thx

**Pull request merged**